src/LuigisBoxBundle/Response/ApiResponse.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Answear\LuigisBoxBundle\Response;
+
+//TODO make this class more usable
+class ApiResponse
+{
+    /**
+     * @var array
+     */
+    private $response;
+
+    private function __construct(array $response)
+    {
+        $this->response = $response;
+    }
+
+    public static function fromArray(array $response): self
```

Comment:
Currently I can't see point of this method & making constructor private

Reply:
I can't see too. I will remove it